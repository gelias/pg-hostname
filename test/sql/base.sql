\set ECHO none
\i sql/hostname.sql
SELECT COALESCE(length(hostname()), 0) >= 0;
