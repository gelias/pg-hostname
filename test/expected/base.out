\set ECHO none
 ?column? 
----------
 t
(1 row)

